contracts/interfaces/IPoolAddressesProvider.sol

```diff
@@ -0,0 +1,250 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
```

Comment:
i think we can keep only needed function in the Aave interface  rather  then keeping them